modules/ROOT/pages/redelivery-policy.adoc

```diff
@@ -0,0 +1,63 @@
+= Redelivery Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, flow, redelivery, policy
+
+A redelivery policy is a filter added to Mule sources, such as modules and connectors, to prevent the dispatch of messages known to cause errors.
+
+When a redelivery policy is added to a flow's source, Mule evaluates the received data before it executes the flow's components. If the data caused failures a specified number of times in the past, the redelivery policy prevents the flow from processing the received data and raises a `REDELIVERY_EXHAUSTED` error. This behavior prevents the runtime from wasting resources by limiting the number of times Mule will execute flow messages that cause errors.
+
+Some connectors have their own logic for handling an error thrown by a redelivery policy. In such cases, check the documentation for the connector to which the source belongs. For example, the `http-connector` might return `429: TOO MANY REQUESTS`.
+
+== Redelivery Policy Configuration Parameters
+
+[%header,cols="20a,20a,20a,60a"]
+|===
+|Field |Type |Default Value |Description
+|*Max Redelivery Count* |Number |5 |Maximum number of times that a message can be redelivered and processed unsuccessfully before raising a `MULE:REDELIVERY_EXHAUSTED` error.
+|*Use Secure Hash* |Boolean |True |Indicates whether to use a secure hash algorithm to identify a redelivered message.
+|*Message Digest Algorithm* |String |SHA-256 a|Secure hashing algorithm to use for the message. If the payload of the message is a Java object (for example, when received by a `vm` listener), Mule ignores the *Message Digest Algorithm* value and returns the value that the payload's `hashCode()` returned.
+|*ID Expression* |Expression|- | Defines one or more expressions that determine when a message has been redelivered. This property can only be set if the value of `Use Secure Hash` is False.
+|*Object Store* |String |An Object Store created by the `ObjectStoreManager`. Non-persistent, with an `entryTtl` of 300 seconds and an expiration interval of 6 seconds. |Object Store where the redelivery counter for each message is stored. You can configure the Object Store as a reference or an inner element. For information, see the xref:connectors::object-store/object-store-connector.adoc[Object Store connector documentation].
+|===
+
+== Redelivery Policy Functionality
+
+The configured redelivery policy works as follows:
+
+Each time the source receives a new message, Mule generates a key from the message by using the configured `Message Digest Algorithm` or by evaluating the ID expression.
+
+* If the processing flow causes an exception, the associated counter is incremented. Once the counter reaches a value bigger than the configured `maxRedeliveryCount` value, Mule throws a `MULE:REDELIVERY_EXHAUSTED` error.
```

Comment:
To help achieve Global English, replace "once" with "after," if that is what you really mean.

"bigger than" > "greater than"